packages/cloudflare/package.json

```diff
@@ -73,7 +73,7 @@
   "dependencies": {
     "@ast-grep/napi": "^0.34.1",
     "@dotenvx/dotenvx": "catalog:",
-    "@opennextjs/aws": "https://pkg.pr.new/@opennextjs/aws@727",
+    "@opennextjs/aws": "https://pkg.pr.new/@opennextjs/aws@0b8705a",
```

Comment:
FYI according to the description, this could be `@731` (each PR is released with the PR number, see the comment on the PR)

Reply:
yeah sorry this was because i pushed different changes to the pr so i just used the commit instead of removing the old download from my pnpm cache. sorted now